.infrastructure/deployment.yml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: todoapp-deployment
+  namespace: mateapp
```

Comment:
The namespace is set to 'mateapp', which is inconsistent with the intended 'todoapp'. Please correct this to ensure consistency across your configuration.